pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -501,6 +503,22 @@ func mutateContainerResource(c *corev1.Container, override *v1beta1.OverrideSpec
 			}
 		}
 	}
+	var found = false
+	for _, logLevel := range override.LogLevel {
+		if logLevel.ContainerName == c.Name {
+			c.Args = append(c.Args, fmt.Sprintf("--v=%d", logLevel.LogLevel))
```

Comment:
Is the double -- required or is one enough? Do all the container entrypoints support --v?

Reply:
i tried installing local version of ConfigSync using single dash `-v=0` and didn't notice any different in the container logs, including git-sync (-v=5). All other args are using double dash so it might be good just to keep it consistent with the others

```
      - args:
        - --root=/repo/source
        - --dest=rev
        - --max-sync-failures=30
        - --error-file=error.json
        ```

Reply:
Yeah I'd agree that a single dash is more conventional for a short option, double dashes for a long option. I'd go with `-v` if the container/binary accepts it

Reply:
changed to single dash `-v`